vividus-plugin-web-app/src/main/java/org/vividus/selenium/BrowserWindowSizeUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.selenium;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class BrowserWindowSizeUtils
+{
+    private BrowserWindowSizeUtils()
+    {
+    }
+
+    public static BrowserWindowSize convertStringToBrowserWindowSize(String sizeAsString)
```

Comment:
move this method to `StringToBrowserWindowSizeParameterConverter`

we can't introduce "utils" classes for each introduced conversion